src/seedu/addressbook/ui/UtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void isAnyNull_noNull() {
+		Object[] testObjects = new Object[] {new String("sdf"), new Integer(123)};
+		boolean result = Utils.isAnyNull(testObjects);
+		assertEquals(result, false);
+	}
+	
+	@Test
+	public void isAnyNull_hasNull() {
+		Object[] testObjects = new Object[] {null, new Integer(123)};
+		boolean result = Utils.isAnyNull(testObjects);
+		assertEquals(result, true);
+	}
+	
+	@Test
+	public void elementAreUnique_isUnique() {
+		ArrayList<Object> testObjects = new ArrayList<Object>();
+		testObjects.add(new String("12313"));
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.